### PR TITLE
Fix Border and BG's

### PR DESCRIPTION
**Pull request by awWhy**
2nd attempt, previous fork didnt wanted to work anymore
If there is lint problem, then I dont know

### CODE REVIEW

File: src/UpdateHTML.ts [hunk 1]
```diff
@@ -615,6 +615,13 @@ export const buttoncolorchange = () => {
         DOMCacheGetOrSet('ascendChallengeBtn').style.backgroundColor = '#171717' :
         DOMCacheGetOrSet('ascendChallengeBtn').style.backgroundColor = 'purple';
 
+    (player.autoAscend && player.cubeUpgrades[10] > 0.5) ?
+        DOMCacheGetOrSet('ascendbtn').style.backgroundColor = 'green' :
+        DOMCacheGetOrSet('ascendbtn').style.backgroundColor = '#171717';
+
+    (player.singularityCount > 0.5)
+    DOMCacheGetOrSet('singularitybtn').style.backgroundColor = '#171717';
```
Comment (KhafraDev) [inline src/UpdateHTML.ts:623]: is this intended?

**Review by KhafraDev — APPROVED** [review]
,

**Review by KhafraDev — CHANGES_REQUESTED** [review]
above comment

File: src/UpdateHTML.ts (hunk 1 above)
Comment (awWhy) [reply src/UpdateHTML.ts:623]: Its makes possible to have no BG while and light mode not lookin weird with Singularity

Comment (awWhy) [reply src/UpdateHTML.ts:623]: image
Here what will happen if to remove these lines

**Comment by awWhy** [comment]
Probably better to have less checks, if there is no reason (Also it need check for p4x5)
If auto Singularity will be added - I left no BG in folder

**Review by KhafraDev — APPROVED** [review]

**Review by KhafraDev — APPROVED** [review]

**Pull request merged**